zebra/zebra_l2.h

```diff
@@ -37,6 +37,7 @@ extern "C" {
 struct zebra_l2info_brslave {
 	ifindex_t bridge_ifindex; /* Bridge Master */
 	struct interface *br_if;  /* Pointer to master */
+	ns_id_t ns_id; /* network namespace where bridge is */
 };
```

Comment:
bridge must belong to the vrf, when used with vrf backend with namespace.
during my test, at one moment, you have the vxlan interface and you look for the bridge interface. having the index is not enough, I think. this is why I added the ns_id, so that the bridge can be found for both vrf backends